packages/client/wallets/aa/front/services/walletService.ts

```diff
@@ -0,0 +1,37 @@
+import { ethers } from 'ethers'
+import { EVMBlockchainIncludingTestnet } from '@crossmint/common-sdk-base'
+import { EVMAAWallet } from '@crossmint/client-sdk-aa';
+import { type TransactionRequest, type TransactionResponse } from "@ethersproject/providers";
+
+
+export const getOrCreateWallet = async () => {
+    const { CrossmintAASDK } = await import("@crossmint/client-sdk-aa");
+    const xm = CrossmintAASDK.init({
+        apiKey: process.env.NEXT_PUBLIC_API_KEY_STG || '',
+    });
+    const signer: ethers.Wallet = new ethers.Wallet("01f4f4a0efe13dbeaff3e2a030f0fa1754e2d90b1391a0120a6a11c1bf10ea6a");
+    const walletInitParams = {
+        signer
+    }
+    const userIdentifier = { email:"_test@test.com" };
```

Comment:
This should be kind of dynamic / or received by input. If not, we will execute the creation flow the first time, and then always the retrieve one

Reply:
**Added two inputs** to add the `email` and the `private key` to create the wallet using `new ethers.Wallet(privateKey);`